src/codecs/webp/lossless_transform.rs

```diff
@@ -32,6 +32,10 @@ impl TransformType {
                 let width = usize::from(width);
                 let height = usize::from(height);
 
+                if image_data.len() < width * height {
+                    return;
+                }
```

Comment:
I don't feel comfortable fully ignoring the transform. It's correct that we can't assume equality in general but should we error _here_  specifically instead?

Reply:
Makes sense. I didn't want to make the fn return a result since it didn't before, but it does seem like an error to have a transform that would be out-of-bounds.